packages/cli/src/bin/index.ts

```diff
@@ -101,9 +95,28 @@ program
       }
     }
 
-    const progressBar = setupProgressBar(
-      runs.length * (dataset.samples || []).length,
+    const multibar = new cliProgress.MultiBar(
+      {
+        clearOnComplete: true,
+        hideCursor: true,
+        forceRedraw: true,
+        format: "{name}: {bar} | {value}/{total} ({percentage}%) | ETA: {eta}s",
+      },
+      cliProgress.Presets.shades_grey,
     );
+    const totalOutputsCount = runs.length * (dataset.samples || []).length;
+    const progressBar = multibar.create(totalOutputsCount, 0, {
+      name: "Outputs",
+    });
+    const totalScoresCount = runs.reduce((agg, run) => {
+      return agg + totalOutputsCount * (run.scorers?.length || 0);
```

Comment:
i think the math is wrong?

```suggestion
      return agg + (dataset.samples || []).length * (run.scorers?.length || 0);
```